content/en/synthetics/guide/browser-tests-using-shadow-dom.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Using Shadow DOM In Your Browser Tests
+kind: guide
+further_reading:
+    - link: '/synthetics/browser_tests'
+      tag: 'Documentation'
+      text: 'Learn about Browser Tests'
+---
+
+## Overview
+
+The Shadow Document Object Model (DOM) API is a web component that allows an encapsulated DOM tree to attach to an HTML element. The [shadow DOM][1] is self-contained and stays isolated from the main document's DOM. 
+
+You may use a shadow DOM for the following use cases:
+
+- Forms and components from third-party libraries
+- Embedded content (such as a video or an image)
+- Chat pop-up integrations
+
+The suite of actions and assertions available in Synthetic browser tests may not be supported for shadow DOM elements.
```

Comment:
I feel we repeat a bit ourselves from l.20 to lign 26. 
Could we sum these 3 paragraphs into 1?